### PR TITLE
[RFR] Fix CORS (Access-Control-Allow-Origin)

### PR DESCRIPTION
**Pull request by rgaidot**

### CODE REVIEW

File: e2e/lib/request.js [hunk 1]
```diff
@@ -20,7 +23,7 @@ export default function myRequest(params, authToken = null, cookies = {}) {
             baseUrl,
             gzip: true,
             json: true,
-            headers: authToken ? { authorization: `${authToken}` } : {},
+            headers: _extend(headers, authToken ? { authorization: `${authToken}` } : {}),
```
Comment (Kmaschta) [inline e2e/lib/request.js]: Can you use `Object.assign({}, object, object2)` instead of `_extend`?
Thanks!

File: config/default.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
 var apiPort = process.env.NODE_PORT || 3000; // eslint-disable-line no-var
-var apiUrl = 'http://localhost:' + apiPort; // eslint-disable-line no-var
-var frontendUrl = 'http://localhost:8080'; // eslint-disable-line no-var
+var apiUrl = `http://localhost:${apiPort}`; // eslint-disable-line no-var
+const frontendUrl = 'http://localhost:8080'; // eslint-disable-line no-var
+const allowUrls = [frontendUrl, apiUrl]; // eslint-disable-line no-var
```
Comment (djhi) [inline config/default.js]: As you are converting the old `var` to `const`, can you apply it to `apiUrl` too and remove the unnecessary eslint comments ?

**Comment by fzaninotto** [comment]
Needs rebase, sorry

**Comment by djhi** [comment]
Superseded by #99

**Pull request closed without merging**